docs/src/_guide/debugging-custom-extractor.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Debug a Custom Extractor
+description: Learn how to debug a custom data extractor.
+layout: doc
```

Comment:
@jared-rimmer let's add a weight so it renders after `Logging and Monitoring` in the left nav.

```suggestion
layout: doc
weight: 25
```